src/components/pages/alerts/AlertsComponent.jsx

```diff
@@ -1,12 +1,76 @@
 import Box from '@material-ui/core/Box';
-
-import SearchBar from './SearchBar';
 import Alert from './Alert';
 
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import DateFnsUtils from '@date-io/date-fns';
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from '@material-ui/pickers';
+
 const AlertsComponent = () => {
+   const [selectedDateFrom, setSelectedDateFrom] = React.useState(new Date());
+   const [selectedDateTo, setSelectedDateTo] = React.useState(new Date());
+
+   const handleDateChangeFrom = (date) => {
+     setSelectedDateFrom(date);
+   };
+
+   const handleDateChangeTo = (date) => {
+     setSelectedDateTo(date);
+   };
+
    return (
       <Box padding="3vh 10vh 3vh 10vh">
-         <SearchBar />
+         <MuiPickersUtilsProvider utils={DateFnsUtils}>
+            <Grid container justify="space-around">
+            <KeyboardDatePicker
+               variant="inline"
+               format="MM/dd/yyyy"
+               margin="normal"
+               id="date-picker-inline"
+               label="Date From"
+               value={selectedDateFrom}
+               onChange={handleDateChangeFrom}
+            />
+            <KeyboardTimePicker
+               variant="inline"
+               margin="normal"
+               id="time-picker"
+               label="Time From"
+               value={selectedDateFrom}
+               onChange={handleDateChangeFrom}
+               KeyboardButtonProps={{
+                  'aria-label': 'change time',
+               }}
+            />
+            <KeyboardDatePicker
+               variant="inline"
+               format="MM/dd/yyyy"
+               margin="normal"
+               id="date-picker-inline"
+               label="Date To"
+               value={selectedDateTo}
+               onChange={handleDateChangeTo}
```

Comment:
I'm not too familiar with the Date object, but using the same function as the time picker would just replace the state with either a time or a date only?

Reply:
This is the first time I use this too. But I think so, since I can only change the time without changing the date on the website and vise versa. I don't know if there are other ways to check that though. I also tried several cases, it seems like when you totally delete the time or date, the other will also be empty, but I think it's acceptable.

Reply:
I searched up methods for the Date object, and it seems like there exist functions like setTime(), setDate(), etc. Have these worked for you?

Reply:
![image](https://user-images.githubusercontent.com/48076141/116804092-fb3e0f80-aad0-11eb-93b4-57410af69ab9.png)
I just tested it through these lines of silly codes and it seems like the time/date does not change while the other is changing. Thus, I think we may not need the setTime() or setDate() specifically. Please tell me if you still we should separate them.

Reply:
I don't think I understand what you mean. Does the time and date print out correctly? I would try printing before setting the date to another format if the time and date aren't being set separately. You might be setting the state to a non-date object before calling those functions on it.

Reply:
![image](https://user-images.githubusercontent.com/48076141/116821487-e2fcdd80-ab2e-11eb-8384-d15aa51e77ea.png)
![image](https://user-images.githubusercontent.com/48076141/116821499-f019cc80-ab2e-11eb-9749-16088ebc14d3.png)
![image](https://user-images.githubusercontent.com/48076141/116821454-c791d280-ab2e-11eb-9243-ed0c82a0330f.png)
For the lines of alert, I just try to print the date and hours to see if they one is changing with the other. I think it did because I can only change one thing as what I did in picture 1 and 2. As for the correctness of the prints, even if the alerts is after the set as my last comment, the time it printed was still the pre-set time instead of the post-set time it was set to. I don't know why but I don't think that's important because I only want to check if one is changing with the other. I moved the prints before the set as you said (picture 3) and it prints out the correct time. I think it means the using the same set function works. 
If you'll be in today's meeting, you can see my demo at that time and please ask me questions.

Reply:
Visually it should work no matter what selectedDateFrom and selectedDateTo is. What those variables are only matters when at some point we have to retrieve that timestamp from our database. I think it makes sense to separate out a time and date function so that the Date object is preserved. We can do that later when those variables are actually needed though :) I can't make it to today's meeting but good work! I'll approve and merge this.